<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ns-client</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/ns-3.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="topology_window">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="log_window">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Noto Sans'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connect_button"/>
   <addaction name="export_button"/>
   <addaction name="run_button"/>
   <addaction name="stop_button"/>
   <addaction name="nodes_button"/>
   <addaction name="connections_button"/>
   <addaction name="tracers_button"/>
   <addaction name="settings_button"/>
  </widget>
  <action name="run_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/run.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>run_button</string>
   </property>
   <property name="toolTip">
    <string>Run simulation</string>
   </property>
  </action>
  <action name="connect_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/connect.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>connect_button</string>
   </property>
   <property name="toolTip">
    <string>Connect to server</string>
   </property>
  </action>
  <action name="export_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/export.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>export_button</string>
   </property>
   <property name="toolTip">
    <string>Export model</string>
   </property>
  </action>
  <action name="stop_button">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/stop.svg</normaloff>:/images/stop.svg</iconset>
   </property>
   <property name="text">
    <string>stop_button</string>
   </property>
   <property name="toolTip">
    <string>Stop simulation</string>
   </property>
  </action>
  <action name="nodes_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/model.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>model_button</string>
   </property>
   <property name="toolTip">
    <string>Model settings</string>
   </property>
  </action>
  <action name="connections_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/connections.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>connections_button</string>
   </property>
   <property name="toolTip">
    <string>Connections list</string>
   </property>
  </action>
  <action name="tracers_button">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/tracer.svg</normaloff>:/images/tracer.svg</iconset>
   </property>
   <property name="text">
    <string>tracers_button</string>
   </property>
   <property name="toolTip">
    <string>Tracers list</string>
   </property>
  </action>
  <action name="settings_button">
   <property name="icon">
    <iconset>
     <normalon>:/images/settings.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>settings_button</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
